assert/_diff.ts

```diff
@@ -95,6 +95,14 @@ export function diff<T>(A: T[], B: T[]): Array<DiffResult<T>> {
     { length: size },
     () => ({ y: -1, id: -1 }),
   );
+  function getPoint(index: number) {
+    const point = fp[index];
+    if (!point) {
+      throw Error(`Missing FarthestPoint at index ${index}`);
+    }
+    return point;
+  }
```

Comment:
Oh, looks like I missed some tests, turns out throwing an error when fp[index] is undefined causes the tests to fail. I need to reconsider how to handle this case.